sisyphus/simple_linux_utility_for_resource_management_engine.py

```diff
@@ -307,6 +312,8 @@ def queue_state(self):
         while True:
             try:
                 out, err, retval = self.system_call(system_command)
+                if retval != 0:
+                    raise subprocess.CalledProcessError(retval, self._system_call_error_warn_msg(system_command))
```

Comment:
`subprocess.CalledProcessError` needs the returncode and the command
```suggestion
                    raise subprocess.CalledProcessError(retval, system_command, self._system_call_error_warn_msg(system_command))
```
or define your own Exception.